### PR TITLE
Activity: Cleanup warnings

### PR DESCRIPTION
**Pull request by tjoyal**

### CODE REVIEW

File: app/javascript/components/rules/activity.js [hunk 1]
```diff
@@ -7,16 +7,17 @@ class Activity extends React.Component {
       <AppProvider>
         <Layout>
           { this.props.events.map((event, i) => {
-            let description = <p>
-              <b>Webhook ID</b><br /><Caption>{ event.identifier }</Caption>
-            </p>
```
Comment (tjoyal) [inline app/javascript/components/rules/activity.js:12]: Nested <p> added by Layout.AnnotatedSection

File: app/javascript/components/rules/activity.js [hunk 2]
```diff
@@ -7,16 +7,17 @@ class Activity extends React.Component {
...
+            </div>
 
             return(
-              <Layout.AnnotatedSection id={ i } title={ event.timestamp } description={ description }>
+              <Layout.AnnotatedSection key={ i } title={ event.timestamp } description={ description }>
                 <Card sectioned>
                   <DescriptionList items={
                     event.details.map((details, j) => {
-                      let badgeStatus = details.level == "info" ? "info" : "warning";
+                      let badgeStatus = details.level === "info" ? "info" : "warning";
```
Comment (tjoyal) [inline app/javascript/components/rules/activity.js:20]: Coersion warning

File: app/javascript/components/rules/activity.js [hunk 3]
```diff
@@ -7,16 +7,17 @@ class Activity extends React.Component {
...
-              <b>Webhook ID</b><br /><Caption>{ event.identifier }</Caption>
-            </p>
+            let description = <div>
+              <b>Webhook ID</b><br />
+              <Caption>{ event.identifier }</Caption>
+            </div>
 
             return(
-              <Layout.AnnotatedSection id={ i } title={ event.timestamp } description={ description }>
+              <Layout.AnnotatedSection key={ i } title={ event.timestamp } description={ description }>
```
Comment (tjoyal) [inline app/javascript/components/rules/activity.js:16]: Elements need to have a `key` attribute

Comment (christianblais) [reply app/javascript/components/rules/activity.js:16]: Makes sense!

File: app/javascript/components/rules/activity.js (hunk 2 above)
Comment (christianblais) [reply app/javascript/components/rules/activity.js:20]: Of course...

File: app/javascript/components/rules/activity.js (hunk 1 above)
Comment (tjoyal) [reply app/javascript/components/rules/activity.js:12]: done!

**Pull request merged**